src/js/angular/similarity/controllers/similarity-list.controller.js

```diff
@@ -107,11 +107,21 @@ function SimilarityCtrl($scope, $interval, toastr, $repositories, $licenseServic
                     data.results.bindings.forEach(function (e) {
                         $scope.usedPrefixes[e.prefix.value] = e.namespace.value;
                     });
+                    $scope.$on('$destroy', function () {
+                        if (yasr) {
+                            yasr.destroy();
+                        }
+                    });
                     yasr = YASR(document.getElementById('yasr'), { // eslint-disable-line new-cap
                         //this way, the URLs in the results are prettified using the defined prefixes
                         getUsedPrefixes: $scope.usedPrefixes,
                         persistency: false,
-                        hideHeader: true
+                        hideHeader: true,
+                        pluginsOptions: {
```

Comment:
I'd encapsulated this in some factory function in an utility or a service because it's repeated a couple of times in different components which is bad.